app/scripts/constants.js

```diff
@@ -676,5 +676,10 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
     'openshift-',
     'kubernetes-',
     'kube-'
-  ]
+  ],
+  MOBILE_CLIENT_VERSION: {
```

Comment:
wondering should this be here or in a mobile service?

Reply:
waiting for this to be merged https://github.com/openshift/origin-web-common/pull/320 so we can follow upstream pattern on versions/resources